### DISCUSSION TITLE
No sound coming in

### DISCUSSION THREAD

**Discussion by kimhio** [ExistentialAudio/BlackHole#546] [Q&A]
Hello,

I am trying to troubleshoot no sound coming into neither Ableton Live nor Garageband after setting everything up as described. 

I've checked my settings again and again, not only against the documentation here but also two different Youtube videos, and I cannot for the life of me see that I am doing anything incorrect. Still no sound is coming in. I do get ominous exclamation marks in my Ableton Preferences window if I select Configure on the individual channel, but not if I go to the same window via the general Preferences drop-down menu (as seen in the screenshot). 

Are there steps I can take to try and eliminate causes? (I'm on macos 10.14.5). Any tips welcome. Thanks!

(I have read the wiki. Apologies for any deviation in terminology...)

<img width="671" alt="Screenshot 2022-03-21 at 12 09 40" src="https://user-images.githubusercontent.com/102028194/159253686-d3e95db7-5410-4bf2-8abc-a1a597c14e6b.png">

<img width="502" alt="Screenshot 2022-03-21 at 12 12 00" src="https://user-images.githubusercontent.com/102028194/159253687-c0f69b06-e4b4-42aa-ba5f-c3fb46ece0c5.png">

<img width="633" alt="Screenshot 2022-03-21 at 12 09 04" src="https://user-images.githubusercontent.com/102028194/159253656-141ea26d-98db-49c6-af72-ceabe731b49c.png">

<img width="635" alt="Screenshot 2022-03-21 at 12 09 09" src="https://user-images.githubusercontent.com/102028194/159253669-49cc9eca-f7c0-4426-a892-6bae9974d642.png">
!

**Comment by MaikWaschfeld** [comment]
Hi,

Some hints.

- If you want to use an aggregated device, you have to select it in LIVE’s Preferences „Audio Input Device“.
- Did you enable all desired input channels in LIVE’s Preferences „Channel Configuration“ > „Input Config"?





With kindest regards…
…Maik Waschfeld

(sent from my MBPro16,2)
also at ***@***.***>

**Reply by kimhio** [reply]
Hello Maik!
And thank you so much for answering. I was just trying out the Aggregated Device, as well as the Multi-Output Device (which is also a type of Aggregated Device as far as I understand), but choosing this over the Multi-Output Device yielded only the same result. 

In the channel configuration window in Live, I can only activate the first 'button', enabling channels 1 & 2 since I'm only using the 2 channel version BlackHole. Which I have done.

But ALSO:

I have a second laptop, a newer one, running macos 10.15.5. I set up everything exactly the same on that computer – running the same version of Live – and there everything WORKS! Which is wonderful! 

But still doesn't answer my question about why it won't work on my older laptop. There is no apparent difference between the two in what I am able to – and choose to – select and enable, neither on the mac system side or in Live, unless there is something I am overlooking.

Still grateful for more tips!

**Comment by MaikWaschfeld** [comment]
![screencaptureA 20220321-140443](https://user-images.githubusercontent.com/8338548/159270508-bb73ed92-e75a-4490-ac14-bc460e850b63.png)
Adding the screenshot to my mail.
Regards...
...Maik

**Comment by MaikWaschfeld** [comment]
Hi,

I’d recommend to install macOS 10.14.6 with the „full installer“ from
<https://apps.apple.com/us/app/macos-mojave/id1398502828?mt=12>.
The installer should be version 14.6.06.

Sometimes applying „full installers“ fixes strange problems.

Besides that, I don’t have any further advice.

What’s the source of your audio signal?


With kindest regards…
…Maik Waschfeld

(sent from my MBPro16,2)
also at ***@***.***>

**Reply by kimhio** [reply]
Thanks, yes that's probably good advice. 

My audio signal is more or less anything that makes sound on my computer. Mostly I was interested in the ability to record any sound from anywhere on the internet, but also from other software. And on the Mac I have where it now works, it works so well. It's amazing to be able to do this so simply. 

Very grateful for the replies – and for BlackHole!
